### PR TITLE
fix(desktop): cannot open camera on desktop

### PR DESCRIPTION
**Pull request by BlackHole1**
by: @crimx

### CODE REVIEW

File: desktop/renderer-app/src/pages/OneToOnePage/index.tsx [hunk 1]
```diff
@@ -302,30 +302,33 @@ export const OneToOnePage = observer<OneToOnePageProps>(function OneToOnePage()
     }
 
     function renderRealtimePanel(): React.ReactNode {
+        console.log("11111", classRoomStore.isRTCJoined);
```
Comment (crimx) [inline desktop/renderer-app/src/pages/OneToOnePage/index.tsx]: delete log

**Review by crimx — CHANGES_REQUESTED** [review]

**Comment by BlackHole1** [comment]
please review again

cc: @crimx

**Review by crimx — APPROVED** [review]

**Comment by BlackHole1** [comment]
@crimx @hyrious Does the flat-web have this problem?

**Comment by hyrious** [comment]
flat-web already has a similar pr #1036 does this work.

**Pull request merged**